CONTRIBUTING.md

```diff
@@ -54,6 +54,11 @@ Try to keep lines of code no longer than 160 characters wide. This isn't a stric
 
 Try to keep your coding style in line with the existing code. It might not exactly match your preferred style but it's better to keep things consistent. StyleCop compliance is enforced through analysis on each build and the treatment of warnings as errors. Any StyleCop settings changes or suppressions must be clearly justified.
 
+## String Formatting
+
+Unless there is a compelling reason not to, for example when serializing data to be parsed later, use the current culture when formatting strings. When string
+formatting methods have an overload that implicitly uses the current culture, opt to use it instead of specifying the culture explicitly.

```

Comment:
> When string formatting methods have an overload that implicitly uses the current culture, opt to use it instead of specifying the culture explicitly

This contradicts our code analysis ruleset. Indeed, when I run code analysis on this branch, I get plenty of these types of warnings:

> Warning	CA1305	Because the behavior of 'string.Format(string, object)' could vary based on the current user's locale settings, replace this call in 'AnyCallCallRule.DescriptionOfValidCall.get()' with a call to 'string.Format(IFormatProvider, string, params object[])'. If the result of 'string.Format(IFormatProvider, string, params object[])' will be displayed to the user, specify 'CultureInfo.CurrentCulture' as the 'IFormatProvider' parameter. Otherwise, if the result will be stored and accessed by software, such as when it is persisted to disk or to a database, specify 'CultureInfo.InvariantCulture'.